CONTRIBUTING.md

```diff
@@ -70,7 +70,10 @@ your documentation changes for clarity, concision, and correctness, as
 well as a clean documentation build.
 
 Write clean code. Universally formatted code promotes ease of writing, reading,
-and maintenance. 
+and maintenance. The Eclipse formatter is installed automatically if [these
+instructions](http://docs.openhab.org/developers/development/ide.html) are used
+when installing Eclipse. Here is [an exported rule set](https://gist.github.com/9037568/7e104f35a7666caf7871cad621c4f0aa)
```

Comment:
I do not like links to non-ESH/openHAB repos here - it can quickly outdate if the formatter settings are adjusted.
In any case I strongly suggest to use the [official Oomph profile](https://github.com/openhab/openhab-distro/blob/master/launch/openHAB2.setup).